### PR TITLE
Update Chinese language files

### PR DESCRIPTION
**Pull request by wcxu21**
Update Chinese language files

### CODE REVIEW

File: DigaoDeskApp/Languages/lang_chinese.json [hunk 1]
```diff
@@ -1,3 +1,299 @@
...
+  "DenyExitByModal": "您无法关闭程序，因为有一个模态表单正在运行",
+  "UntrackedTag": "untracked",
+
+  "BtnOK": "确定",
+  "BtnCancel": "取消",
+
+  "CountLabel": "计数：{0}",
+
+  "Config_Title": "设置 - Digao Desk",
+  "Config_Tab_Logs": "日志",
```
Comment (digao-dalpiaz) [inline DigaoDeskApp/Languages/lang_chinese.json:20]: Missing "Config_Tab_General" var

File: DigaoDeskApp/Languages/lang_chinese.json [hunk 2]
```diff
@@ -1,3 +1,299 @@
...
+  "Config_Tab_Applications": "应用程序",
+  "Config_Tab_Repositories": "Git 存储库",
+  "Config_Logs_Font": "字体",
```
Comment (digao-dalpiaz) [inline DigaoDeskApp/Languages/lang_chinese.json:25]: Missing:
Config_General_Language
Config_General_LanguageInfo

File: DigaoDeskApp/Languages/lang_chinese.json [hunk 3]
```diff
@@ -1,3 +1,299 @@
...
+  "Config_Repos_ShellProgram": "外壳程序",
+  "Config_Repos_DiffProgram": "差异程序",
+  "Config_Repos_DiffProgramArgs": "参数",
+  "Config_Repos_DiffProgramArgsInfo": "[old] = 旧文件 / [new] = 新文件",
+  "Config_Repos_NewBranchPrefixList": "新分支前缀列表",
+  "Config_Repos_NewBranchPrefixListInfo": "CTRL+ENTER = 新行",
+  "Config_Repos_AutoFetch": "远程操作中的自动获取",
+  "Config_Repos_CommitMsg": "提交信息",
+  "Config_Repos_CommitMsgInfo": "使用 [branch] 作为分支名称",
+
```
Comment (digao-dalpiaz) [inline DigaoDeskApp/Languages/lang_chinese.json:50]: Missing:
Config_InvalidLanguage
Config_GitRepositoryNotFound

**Review by digao-dalpiaz — CHANGES_REQUESTED** [review]
Hello, please review some missing tags

**Comment by wcxu21** [comment]
Update Chinese translation

**Review by digao-dalpiaz — APPROVED** [review]
ok

**Pull request merged**